cmd/controller/controller.go

```diff
@@ -337,6 +348,58 @@ func ParseAndValidateConfig(v *viper.Viper) (*config.Config, error) {
 	return cfg, nil
 }
 
+// Viper downcases keys in maps used in inputs, but in the case of resource classes where hugepages are used, it
+// gets downcased to "hugepages-2mi". This function fixes that by recapitlizing the "Mi" part for any hugepages resource
+// claims. This is important because the hugepages resource name must be capitalized as "hugepages-2Mi" (or similar),
+// otherwise k8s won't recognize the resource claim.
+func recapitalizeHugePagesResourceClasses(rc *config.ResourceClass) error {
```

Comment:
Is this possible to move these to another file? `recapticalize_resource.go?` `controller.go` is getting big. And this logic is quite isolated and narrow.

Reply:
i solved this by moving the flags to their own file, as "what flags we have" and "how we process their values" feel like two separate things to me